bin/git-mpush

```diff
@@ -225,11 +236,12 @@ sub run {
 
   # Graft the temp branch on the shared branch
   info("Grafting new commits to branch '$temp_branch'.");
+  info("Command run: git rebase --onto $shared_branch $remote/$faster_branch $temp_branch");
   if (!cmd_ok("git rebase --onto $shared_branch $remote/$faster_branch $temp_branch")) {
     info("Action failed, cleaning up");
     cmd("git rebase --abort");
     switch_branch($current_branch, 1);
-    cmd("git branch -D $temp_branch");
+    #cmd("git branch -D $temp_branch");
```

Comment:
Are you sure you dont want to delete this temporary branch, i think we should since it is a cleaning process and trying to revert everything back to what it was before.